region/norfair/east/Double Chamber.json

```diff
@@ -464,6 +499,53 @@
         }
       }
     },
+    {
+      "link": [2, 4],
+      "name": "Tricky Spring Ball Jump",
+      "entranceCondition": {
+        "comeInJumping": {
+          "minTiles": 2,
+          "speedBooster": "any"
+        }
+      },
+      "requires": [
+        "canPrepareForNextRoom",
+        "canLateralMidAirMorph",
+        "canTrickySpringBallJump",
+        "canTrickyJump",
+        {"heatFrames": 570}
+      ],
+      "note": [
+        "Spin jump into the room with at least 2 tiles of run speed.",
+        "Perform a lateral mid-air morph into a spring ball jump.",
+        "Land on the second moving platform."
+      ]
+    },
+    {
+      "link": [2, 4],
+      "name": "Tricky Dash Jump",
```

Comment:
And I thought these would be HiJump+Speed or just Spacejump strats to go more directly from 2 to 4.

Reply:
Hmm yeah there are a lot more strats that can be added here. Working on it now

Reply:
Ok, added a bunch more. Also redid my heat frames. I had made a silly mistake of looking at the RTA timer (on collecting the item or leaving the room), but for the spring ball jump strats this was very wrong since it was counting time in the pause menu.